src/Classes/Providers/ParcelProvider.ps1

```diff
@@ -320,9 +320,9 @@ class ParcelProvider
         return [string]::Empty
     }
 
-    [string] GetSourceArgument([ParcelPackage]$_package)
+    [array] GetSourceArgument([ParcelPackage]$_package)
     {
-        return [string]::Empty
+        return [array]::Empty
     }
```

Comment:
`[string]` and `[string]::Empty` 👀

Reply:
Yes?\

Reply:
Not sure if it's the diff playing up; but I think the function's return type still says `[array]` ?

Reply:
fixed?